### ISSUE TITLE
Cannot read noColor of null. CLI Reporter > NodeJs Module

### ISSUE THREAD

**Issue by derpoho** [postmanlabs/newman#859]
1. Newman Version (can be found via `newman -v`): > 3.3.1
 2. OS details (type, version, and architecture): MacOs
 3. Are you using Newman as a library, or via the CLI? Library -> NodeJS
 3. Did you encounter this recently, or has this bug always been there: Since 3.4.0
 4. Expected behaviour: Run Collection

This Issue seems to be regarding this commit: https://github.com/postmanlabs/newman/commit/acc7ddf6b7be5c191c045392334aeedd35d67eae

The Error thrown is
```
TypeError: Cannot read property 'noColor' of undefined
    at Object.inspector (~/node_modules/newman/lib/reporters/cli/cli-utils.js:109:37)
    at new PostmanCLIReporter (~/node_modules/newman/lib/reporters/cli/index.js:52:28)
```

After looking into this. It's first introduced with the above commit at Version 3.4.0. Version 3.3.1 works fine for me. When i log the runOptions (https://github.com/postmanlabs/newman/commit/acc7ddf6b7be5c191c045392334aeedd35d67eae#diff-7f96765916328a1d5ffea0fddaa42464R106) then they are null.

Thanks guys!

Edit: Maybe i should mention @kunagpal as Committer of this.

**Comment by derpoho** [comment]
Further Investigation.

The Error starts here: https://github.com/postmanlabs/newman/blob/acc7ddf6b7be5c191c045392334aeedd35d67eae/lib/reporters/cli/index.js#L52

When i submit an empty object as option for noColor, it works, just because the object is there ;)

```
newman.run({
    collection: 'https://api.getpostman.com/***',
    environment: envVars,
    reporters: 'cli',
    noColor: {} // Without this, the CLI Reporter fails
}, function (err) {
    if (err) { throw err; }
    console.log('collection run complete!');
});
```

**Comment by shamasis** [comment]
@czardoz fyi

**Comment by czardoz** [comment]
@derpoho The fix is out with Newman v3.4.2 :)
